src/edu/psu/sweng500/team8/gui/BoardGUI.java

```diff
@@ -0,0 +1,261 @@
+package edu.psu.sweng500.team8.gui;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+import edu.psu.sweng500.team8.coreDataStructures.Cell;
+import edu.psu.sweng500.team8.coreDataStructures.CellCoordinates;
+import edu.psu.sweng500.team8.play.GameSession;
+
+public class BoardGUI extends JPanel {
+	
+	private static final int BOARD_SIZE = 486;
+	private BlockGUI[][] blocks;
+	private GameSession gameSession;
+	private CellGUI selectedCell;
+	private Set<CellGUI> highlightedIncorrectCells = new HashSet<CellGUI>();
+	
+	public BoardGUI() {
+		
+		setPreferredSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
+		setMaximumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
+		setMinimumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
+
+		blocks = new BlockGUI[3][3];

```

Comment:
See my other comment below. I know it's not required, but it's preferred for readability.